### DISCUSSION TITLE
Site "Recursos front end"

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#1373] [Dicas]
Recentemente, lançamos o site "Recursos front end", que pretende ser uma coletânea dos melhores recursos, com tutoriais, guias, ferramentas etc.

Venho à comunidade fazer esse comunicado e pedir a opinião dos colegas a respeito. Para os que se interessarem, por favor, entrem, deem uma olhadinha e digam o que acharam.

Desde já, agradecemos! 😁

https://recursosfrontend.desenvolvimentoparaweb.com/

PS: Curiosidade: feito com Brunch + Preact + Firebase.

**Comment by nandomoreirame** [comment]
Tópe!

**Comment by nandomoreirame** [comment]
Esse formulário torto que me deu um ruim hahahhahaha 😂

**Comment by rafaelnogueira1** [comment]
Cara que irado parabéns!!
Pow na parte de Javascript também cabia colocar o Javascript Eloquente ou o You don´t know JS que estão aí no github pro povo.

**Comment by gabrielbs** [comment]
Parabéns @tarciozemel legal demais :D 

Só o formulário que realmente me deu uma tonturinha (serio)

**Comment by tapmorales** [comment]
Nossa. Curti demais. Parabens.

**Comment by rubenmarcus** [comment]
no de Performance tambem não vai o do google? https://developers.google.com/speed/pagespeed/insights/

**Comment by tarciozemel** [comment]
> no de Performance tambem não vai o do google? https://developers.google.com/speed/pagespeed/insights/

O GTmetrix mostra a análise do PSI, mas não tenho certeza se já atualizaram para pegar as métricas do Lighthouse também. Vamos conferir e, conforme for, fazer uma atualização.

Obrigado pela dica! 👍

**Comment by simaodeveloper** [comment]
> Esse formulário torto que me deu um ruim hahahhahaha 😂

Buguei tambem asuhaushaush

**Comment by jaocarli** [comment]
Até salvei aqui nos favoritos! Muito bom!
